firmware/main.c

```diff
@@ -553,17 +553,17 @@ static void cmd_motor_rt(BaseSequentialStream *chp, int argc, char *argv[]){
                         if(++i >= argc) continue;
                         mOpt = argv[i];  // Selects the operation mode
                 }
-		else if(strcmp(argv[i], "-s") == 0) {
```

Comment:
Again, in this commit, there is a bunch of commented out code. Like I wrote before, if it's obsolete, remove it; if it's optional make it optional.